even_fib_nums.js

```diff
@@ -5,22 +5,43 @@
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
+
 function _sumFibs( maxFibValue ) {
   let sum = 0;
 
-  // do your work here
-
+  let fibA = 0;
+  let fibB = 1;
+  let fibSum = 1;
```

Comment:
This variable could be named better. Since you already have a sum variable, this variable would be better named as fibCurrent.